lint-configs/eslint-rules/i18n-import-location.mjs

```diff
@@ -35,8 +35,7 @@ export default {
             fix(fixer) {
               const canSafelyReplace =
                 node.specifiers.length === 1 &&
-                node.specifiers[0].type === "ImportDefaultSpecifier" &&
-                node.specifiers[0].local.name === "i18n";
```

Comment:
We'll need to update the `existingImport` check on line 44 to handle a different `localName` as well.

(or... maybe we can just drop that existing-import-manipulation now that we have the no-duplicate-imports fixer?)